### ISSUE TITLE
Proposal: stress test

### ISSUE THREAD

**Issue by vctrferreira** [uw-labs/bloomrpc#68]
So I think that add one feature to stress test or something like that can be a great acquisition for bloomRPC, what you think about that @fenos?

**Comment by fenos** [comment]
@vctrferreira Hi! 

This sounds an interesting proposal. Could you elaborate more on what sort of functionality you would expect from this stress tool? thanks

**Comment by vctrferreira** [comment]
I think that apache bench is one option to reflect features.

So I think that is great feature if we had some options/details like that:

* How to set number of messages and number o clients sending simultaneous request and show basically response time of then and if had any error or something like that.
* The average of  response time.

I don't know much of benching mark tooling but I think that it's the basic.

**Comment by fenos** [comment]
@vctrferreira I gave some thoughts about this during the past week. 

I came to the conclusion that BloomRPC it's a gRPC client not a stress test tool.

I believe that starting supporting stress test functionality might deviate the direction of BloomRPC which aim's to bring the best experience to query gRPC service. 

I probably envision BloomRPC in the future to support some sort of plugin system, which the stress tool could definitely be one of them.

For now, I'm closing the issue and I encourage the community to come out with a stress test tool for gRPC if it doesn't exist yet.
